pdns/dnsdistdist/docs/changelog.rst

```diff
@@ -1,6 +1,394 @@
 Changelog
 =========
 
+.. changelog::
+  :version: 1.6.0-alpha1
+  :released: 2nd of February 2021
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 9273
+
+    Add Lua bindings to get a server's latency
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 9225
+
+    Wrap more FILE objects in smart pointers
+
+  .. change::
+    :tags: Bug Fixes
+    :pullreq: 9222
+    :tickets: 9075
+
+    Use toStringWithPort instead of manual addr/port concat (Mischan Toosarani-Hausberger)
+
+  .. change::
+    :tags: Bug Fixes
+    :pullreq: 9275
+
+    Force a reconnection when a downstream transitions to the UP state (Nuitari, Stephane Bakhos)
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 9049
+
+    Set the default EDNS buffer size on generated answers to 1232
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 9157
+    :tickets: 9156
+
+    Add support for FreeBSD's SO_REUSEPORT_LB
+
+  .. change::
+    :tags: Improvements, Performance
+    :pullreq: 9382
+
+    Speed up the round robin policy
+
+  .. change::
+    :tags: Bug Fixes
+    :pullreq: 9381
+
+    Handle EINTR in DelayPipe
+
+  .. change::
+    :tags: Improvements, Performance
+    :pullreq: 9424
+
+    Avoid unnecessary allocations and copies with DNSName::toDNSString()
+
+  .. change::
+    :tags: Improvements, Performance
+    :pullreq: 9420
+    :tickets: 8993
+
+    Get rid of allocations in the packet cache's fast path
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 9428
+
+    Accept string in DNSDistPacketCache:expungeByName
+
+  .. change::
+    :tags: Bug Fixes
+    :pullreq: 9431
+
+    Handle empty DNSNames in grepq()
+
+  .. change::
+    :tags: New Features
+    :pullreq: 9175
+
+    Add per-thread Lua FFI load-balancing policies
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 9466
+
+    DNSName: add toDNSString convenience function
+
+  .. change::
+    :tags: Improvements, Security
+    :pullreq: 8969
+
+    Use more of systemd's sandboxing options when available
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 8993
+    :tickets: 5131
+
+    Skip EDNS Cookies in the packet cache
+
+  .. change::
+    :tags: Improvements, DNS over HTTPS, DNS over TLS
```

Comment:
the only valid tags are `conf.py:changelog_sections = ['New Features', 'Improvements', 'Bug Fixes', 'Removals']`, the rest would be ignored, as far as I know

Reply:
We have been setting values from `changelog_inner_tag_sort = ['Security', 'DNS over HTTPS', 'DNS over TLS', 'DNSCrypt', 'DNSTAP', 'Protobuf', 'Performance', 'Webserver', 'Metrics']` for a while now, though?

Reply:
Huh, I had no idea. I also don't see it reflected in the changelog HTML, but I suppose it affects sorting?

Reply:
Looks like it. Sounds pretty useless in retrospect, but since the tags are already there.. :)